app/Command/REPL.hs

```diff
@@ -347,14 +352,28 @@ command = loop <$> options
                           liftIO $ shutdown state
                         Right (Just c') -> handleCommandWithInterrupts state c'
 
+                    loadUserConfig :: state -> StateT PSCiState (ReaderT PSCiConfig IO) ()
+                    loadUserConfig state = do
+                      configFile <- (</> ".psci") <$> liftIO getCurrentDirectory
+                      exists <- liftIO $ doesFileExist configFile
+                      when exists $ do
+                        ls <- lines <$> liftIO (readUTF8File configFile)
+                        for_ ls $ \l -> do
+                          liftIO (putStrLn l)
+                          case parseCommand l of
+                            Left err -> liftIO (putStrLn err >> exitFailure)
+                            Right cmd@Import{} -> handleCommand' state cmd
+                            Right _ -> liftIO (putStrLn "Only import declarations are supported in the .psci file" >> exitFailure)
```

Comment:
Might be nicer to print a warning, ignore declarations which aren't imports, and continue?